packages/cmd/src/services/cache/presets.ts

```diff
@@ -85,7 +85,7 @@ async function dumpPwConfigForGitlab(
     config.presetOutput ?? PRESET_OUTPUT_PATH,
     `EXTRA_PW_FLAGS="${pwCliOptions}"
 EXTRA_PWCP_FLAGS="${lastFailedOption}"
```

Comment:
We absolutely need to keep them on variables that contain spaces, otherwise `source` will try to execute the code after the space. But they are safe to remove for simple booleans or numbers.